app/views/games/show.html.erb

```diff
@@ -1,13 +1,27 @@
-
 <br />
-<div class="booyah-box col-xs-10 col-xs-offset-1 text-center">
-<p><h1><%= @game.name %></h1></p>
+<!-- <div class="booyah-box col-xs-10 col-xs-offset-1 text-center"> -->
+<p><h1><center><%= @game.name %></center></h1></p>
 <br />
 </div>
 
 <!-- Table version of game board - rest of code in games.scss -->
 
 <table>
+
+<% 8.times do |row_index| %>
+  <tr>
+
+    <% 8.times do |col_index|  %>
+      <td>
+
```

Comment:
Add a helper method in games_helper

```
def piece_by_position(game, x, y)
  piece = game.piece_at(x, y)
  if piece.nil?
    return ""
  else 

end
```

Add `<%= game.piece_at(row_index, col_index) %>`